src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -376,6 +377,7 @@ const formConfig = {
           uiSchema: {},
           schema: { type: 'object', properties: {} },
         },
+        ...form0781PagesConfig,
```

Comment:
I believe the new 0781 flow is supposed to appear after the prisonerOfWar section.

Reply:
updated. This places it after toxic exposure.